packages/core/src/App/Components/Elements/Modals/RedirectNotice/redirect-notice-modal.jsx

```diff
@@ -53,20 +53,18 @@ const RedirectNoticeModal = ({ is_logged_in, is_eu, portal_id }) => {
     }, [is_logged_in, is_eu]);
 
     return ReactDOM.createPortal(
-        dialog_status ? (
```

Comment:
Question: We dont need the condition ?

Reply:
It's unnecessary to include that because it's just checking if the dialog is true. It doesn't make a difference even when I exclude it.

Reply:
@aizad-deriv this condition should remain because this is needed to close the redirection modal when a user clicks Cancel, no?:
```
const onCancelDialog = () => {
        setDialogStatus(false);
    };
```

Reply:
@maryia-deriv Yes it exist in the file it is passed to the `onCancel` prop for the `Dialog` component @ line 62

Reply:
@aizad-deriv correct, so when a user clicks Cancel, the dialog_status is changed to false, and the redirection modal disappears, i.e. we end up with `null`. Should it work like this? if yes, we have to restore the condition🙂

Reply:
@maryia-deriv that's true I will restore the condition. Thank you 🙏🏼